core/ledger/kvledger/tests/v1x_test.go

```diff
@@ -107,18 +82,45 @@ func TestV13WithStateCouchdb(t *testing.T) {
 	dataHelper.verify(h2)
 
 	// drop couchDB and other rebuildable dbs to test db rebuild
-	dropCouchDBs(t, couchdbConfig)
+	require.NoError(t, statecouchdb.DropApplicationDBs(couchdbConfig))
```

Comment:
Not in this PR but just noticed - though we are not writing anything in parallel here, still better to move this line after the next line (i.e., after closing the ledger).